src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+interface Props {
+  user: User | null;
```

Comment:
handle it in the upper component
```suggestion
  user: User;
```